dev/env/opensource/development.env

```diff
@@ -2,3 +2,19 @@ APP_MODE=full
 APP_ENV=development
 APP_INSTALLATION=opensource
 APP_DEV_MODE=1
+
+HC=1
+
+ZITADEL=false
+
+CLIENT_ID=269829931184226314@datalens
+CLIENT_SECRET=GRCfRsteLvSqolF122loogYU2oa7LhufWTdS6WCmiOcmSH48rqTmZOKAtLB0A4YT
+
+ZITADEL_PROJECT_ID=269829807099871242
+
+ZITADEL_URI=http://localhost:8085
+APP_HOST_URI=http://localhost:8080
```

Comment:
In development mode, the application uses a different port. Why do we need this?

Reply:
agree with port, changed to 3030. APP_HOST_URI is used by Zitadel to logout. It is a part of the contract (post_logout_redirect_uri param). It should know where to address me back after logout. Moreover, host uri must be equal host uri configured in Zitadel for redirection :     

```
        `${zitadelUri}/oidc/v1/end_session?` +
        new URLSearchParams({
            post_logout_redirect_uri: appHostUri + '/auth',
            client_id: clientId,
        });
```